2200021/10505.py

```diff
@@ -0,0 +1,17 @@
+# 소득 불균형
+
+import sys
+sys.stdin = open('input.txt','r')
+input = sys.stdin.readline
+
+for t in range(1,int(input())+1):
+    n = int(input())
+    nums = list(map(int,input().split()))
+    average = sum(nums)//n
```

Comment:
```suggestion
    average = sum(nums)/n
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄